src/build.rs

```diff
@@ -322,7 +323,22 @@ pub fn cbuild(
 
     if cur_hash.is_none() || prev_hash != cur_hash {
         build_def_file(&ws, &name, &rustc_target, &root_output)?;
-        build_implib_file(&ws, &name, &rustc_target, &root_output)?;
+
+        let mut dlltool = PathBuf::from("dlltool");
```

Comment:
this can be
``` rust
let dlltool = std::env::var_os("DLLTOOL").unwrap_or_else(|| PathBuf::from("dlltool"))
```

Reply:
Nope did not work. Type mismatch. Any idea how to fix that in the one liner or should i just revert it to the old version?